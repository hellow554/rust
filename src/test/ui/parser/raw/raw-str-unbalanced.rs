static s: &'static str =
    r#" //~ ERROR too many `#` when terminating raw string
      "##;

fn main() {}
