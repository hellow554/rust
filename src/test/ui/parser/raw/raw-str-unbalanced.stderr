error: too many `#` when terminating raw string
  --> $DIR/raw-str-unbalanced.rs:2:5
   |
LL |       r#"
   |       ^- The raw string has 1 leading `#`...
   |  _____|
   | |
LL | |       "##;
   | |________-^
   |          |
   |          ...but is closed with 2.
   = help: remove the unneeded `#`

error: aborting due to previous error

